src/App.tsx

```diff
@@ -17,18 +40,40 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                setQuery={setQuery}
+                setStatus={setStatus}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+
+              {!todos.length &&
+                <p>No toods found
+                </p>
+              }
```

Comment:
```suggestion
              {!preparedTodos.length &&
                <p>No toods found</p>
              }
```

I guess it should be visible if no one todo matches the filter

Reply:
I'd also name it `filteredTodos` or `visibleTodos` (not critical)